### PR TITLE
AV-162992 add alternate cert to avi controller with a different name

### PR DESCRIPTION
**Pull request by arihantg**
add alternate cert to avi controller with a different name than regular cert

### CODE REVIEW

File: internal/lib/lib.go [hunk 1]
```diff
@@ -551,6 +551,9 @@ func GetTLSKeyCertNodeName(infrasetting, sniHostName, secretName string) string
 	if infrasetting != "" {
 		namePrefix += infrasetting + "-"
 	}
+	if strings.HasSuffix(secretName,"-alt"){
```
Comment (apalsule) [inline internal/lib/lib.go:554]: can you fix this golangci-lint issue ?

Comment (arihantg) [reply internal/lib/lib.go]: fixed

**Comment by arihantg** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Review by aaha97 — APPROVED** [review]
LGTM

**Pull request merged**